src/style.css

```diff
@@ -1,3 +1,147 @@
-body {
+html body {
+  font-family: Roboto, Arial, sans-serif;
+  --header-menu-active-color: rgba(0, 172, 220, 1);
+  --menu-apple-width: 37px;
+  --blue-underline-height: 4px;
+  --blue-underline-redius: 30%;
+  --total-length-horizontal-margins-inside-header: 140px;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+}
+
+.logo {
+  position: relative;
+  left: 50px;
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__link {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo__link img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+.nav {
+  position: relative;
+  right: 50px;
+}
+
+.nav__list {
+  font-size: 12px;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  width: calc(
+    var(--menu-apple-width) +
+    58px +
+    88px +
+    138px +
+    54px +
+    51px +
+    40px +
+    35px +
+    var(--total-length-horizontal-margins-inside-header)
+  );
+}
+
+.nav_item {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 15px;
+  margin-left: 20px;
+}
+
+.nav_item.apple {
+  margin-left: 0;
+}
+
+.is-active {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 100% 0;
+  color: rgba(0, 0, 0, 1);
+}
+
+.is-active.apple {
+  color: var(--header-menu-active-color);
+  width: var(--menu-apple-width);
+}
+
+.is-active.samsung {
+  width: 58px;
+}
+
+.is-active.smartphones {
+  width: 88px;
+}
+
+.is-active.laptops&computers {
+  width: 138px;
+}
+
+.is-active.gadgets {
+  width: 54px;
+}
+
+.is-active.tablets {
+  width: 51px;
+}
+
+.is-active.photo {
+  width: 40px;
+}
+
+.is-active.video {
+  width: 35px;
+}
```

Comment:
In your HTML there are no classes .video .photo etc. This code is redundant and you dont need to set width to all elements separately. Let them to take as much width as they need. Remove this styles